queue_test.go

```diff
@@ -309,3 +291,15 @@ func CheckNumMessages(t *testing.T, queueName string, want int) {
 		}
 	}
 }
+
+func TestMain(m *testing.M) {
```

Comment:
you don't need it anymore, set env variables in tests. which need it

Reply:
we need it in every test so is it not better to set it once ??